### PR TITLE
Change UI design

### PR DESCRIPTION
**Pull request by imsuviiix**
캡처
image

위와 같이 UI design 수정했습니다.

### CODE REVIEW

**Comment by imsuviiix** [comment]
그리고, 유저가 선택할 기사 수 8개로 늘렸고, 4번 선택하면 logits 값 반환하게 하는 것이 
유저 입장에서도 덜 지루하고 좋을 것 같습니다! (우선 시연용으로)
